lua/neotest-busted/logging.lua

```diff
@@ -16,6 +16,7 @@ local function log(level, context, message, ...)
     local args = { ... }
 
     vim.schedule(function()
+        if not unpack(args) then return end
```

Comment:
A better approach would be to just move assignment to `formatted_message` out of the callback to `vim.schedule` and make it `local formatted_message = message:format(…)`.

Reply:
#33

Reply:
👍